docs/cn/Running-Alluxio-on-GCE.md

```diff
@@ -20,7 +20,10 @@ priority: 4
 
 安装Google Vagrant 插件:
 
-{% include Running-Alluxio-on-GCE/install-google-vagrant-plugin.md %}
```

Comment:
Can you check all these imported snippets? If they are no longer referenced, then please remove them as well.

Reply:
I have update the en docs and removed docs/_includes/Running-Alluxio-on-GCE.
Please check it.